vortex-serde/Cargo.toml

```diff
@@ -19,15 +19,23 @@ arrow-buffer = { workspace = true }
 arrow-schema = { workspace = true }
 bytes = { workspace = true }
 croaring = { workspace = true }
+compio = { workspace = true, features = ["bytes", "macros"], optional = true }
 flatbuffers = { workspace = true }
-futures = { workspace = true }
+flume = { workspace = true }
+futures = { workspace = true, features = ["std"] }
```

Comment:
i forget why but i had to add the `std` flag to get access to some type that requires an allocator, i forget which